task/06-conditions-n-loops-tasks.js

```diff
@@ -211,7 +229,14 @@ function findFirstSingleChar(str) {
  *
  */
 function getIntervalString(a, b, isStartIncluded, isEndIncluded) {
-    throw new Error('Not implemented');
+    let interval = '';
+
+    isStartIncluded ? interval += '[' : interval += '(';
+    a > b ? interval += b + ', ' + a : interval += a + ', ' + b;
+    isEndIncluded ? interval += ']' : interval += ')';
```

Comment:
avoid this code style, if-else is more readable than multiple conditional operators ? :